docs/custom-flows/embedded-email-links.mdx

```diff
@@ -1,46 +1,105 @@
+- Promotional emails for users.
+- Recovering abandoned carts.
+- Surveys or questionnaires.
 
-## Creating email links
+This guide will teach you how to set this flow up.
 
-Embeddable email links leverage [sign-in tokens](https://clerk.com/docs/reference/backend-api/tag/Sign-in-Tokens#operation/CreateSignInToken) which can be created from your backend.
+<Steps>
+
+### Generate a sign-in token
+
+Embeddable email links leverage [sign-in tokens](https://clerk.com/docs/reference/backend-api/tag/Sign-in-Tokens#operation/CreateSignInToken), which are JWTs that can be used to sign in to an application without specifying any credentials. A sign-in token can be used at most once, and can be consumed from the Frontend API using the [`ticket`](/docs/references/javascript/sign-in/sign-in#sign-in-create-params) strategy.
+
+> [!NOTE]
+> By default, sign-in tokens expire in 30 days. You can optionally supply a different duration in seconds using the `expires_in_seconds` property.
+
+The following example demonstrates a cURL request that fetches a valid sign-in token: 
 
 ```bash
-curl --request POST \
-  --url 'https://api.clerk.com/v1/sign_in_tokens' \
-  -H 'Authorization: Bearer {{bapi}}' \
+curl 'https://api.clerk.com/v1/sign_in_tokens' \
+  -X POST \
+  -H 'Authorization: Bearer {{secret}}' \
   -H 'Content-Type: application/json' \
-  -d '{
-    "user_id": "user_28dVmJleKwFWrefTJN7skmrbtJi",
-  }'
+  -d '{ "user_id": "user_123" }'
 ```
 
-This will return a token, which can then be embedded as a query param in any link, something like:
+This will return a token, which can then be embedded as a query param in any link, such as the following example:
 
 `https://your-site.com/welcome?token=THE_TOKEN`
 
-Once you have the link, this can be embedded anywhere, such as an email.
+You can embedded this link anywhere, such as an email.
+
+### Detect tokens in email links
+
+To do something with the email links you generate, you must setup a page in your frontend that detects the sign-in token, signs the user in, then performs whatever actions you want.
+
+The following example demonstrates basic code that detects a token and uses it to initiate a sign-in with Clerk:
+
+<Tabs items={["Next.js"]}>
+<Tab>
+<CodeBlockTabs type="framework" options={["App", "Pages"]}>
+```jsx filename="app/accept-token/page.jsx"
+"use client";
+import { useUser, useSignIn } from "@clerk/nextjs";
+import { useEffect, useState } from "react";
+import { useSearchParams } from "next/navigation";
+
+export default function AcceptToken() {
+  const { signIn, setActive } = useSignIn();
+  const { user } = useUser();
+  const [signInProcessed, setSignInProcessed] = useState<boolean>(false);
+  const signInToken = useSearchParams().get("token");
+
+  useEffect(() => {
+    if (!signIn || !setActive || !signInToken) {
+      return;
+    }
+
+    const aFunc = async () => {
+      try {
+        // Create a signIn with the token, note that you need to use the "ticket" strategy.
```

Comment:
It's too long for the code block width.

Also, I think you'll have to make these changes to the Pages Router example too

```suggestion
        // Create a signIn with the token
        // Note that you need to use the "ticket" strategy.
```